### ISSUE TITLE
Detect if connection has been throttled

### ISSUE THREAD

**Issue by jstejada** [mscdex/node-imap#593]
Is there a way to detect if the connection is being throttled?

i.e. when I activate debug logs, I see the following:

```
<= 'A2 OK Success [THROTTLED]'
```

which indicates that the account has been throttled. is there a way to detect this via node-imap?

**Comment by mscdex** [comment]
Not really. Since that is more or less a freeform text field and there is no standard way to detect throttling, you're pretty much out of luck.
